common/major.py

```diff
@@ -14,16 +14,17 @@ def get_majors() -> dict[str, str]:
     Returns:
         dict[str, str]: The dictionary of major short codes to major full text.
     """
+    return {}
```

Comment:
What is this for?

Reply:
I got tired of waiting for it on start up 💀 will revert